include/mbedtls/mbedtls_config.h

```diff
@@ -2160,6 +2160,14 @@
  */
 #define MBEDTLS_DEBUG_C
 
+/**
+ * \def MBEDTLS_INST
+ *
+ * Enable the instrumentation functions to report invocations of various
+ * primitives. See 'include/mbedtls/inst.h' for more information.
+ */
+//#define MBEDTLS_INST
```

Comment:
The trace is very verbose, so I wonder if it should be disabled in the `full` configuration? (See `EXCLUDE_FROM_FULL` in `config.py`.) If so it should have its own dedicated component in `tests/scripts/all.sh` (like `MBEDTLS_MEMORY_DEBUG`). Given that this feature is meant to be used with TLS, and that there's history of it breaking programs used in TLS testing, this component should run a subset of `ssl-opt.sh` in addition to the unit tests.

Reply:
Yes, that makes more sense. I wasn't aware of that configuration mechanism, and need to familiarize myself with the architecture of `all.sh` & `config.py`.

Reply:
I added it to `EXCLUDE_FROM_FULL`, but I'm unfamiliar with jenkins scripting, so I won't have a test entry in this proposed PR.